File: Basic Coding Problems/01 Fundamentals/007 Write a program to enter length in centimeter and convert it into meter and kilometer/C++/centimeter_to_meter_kilometer.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main() {
    int length_in_cm;
    float length_in_m, length_in_km;

    cout << "Enter the length in centimeter: ";
    cin >> length_in_cm;

    length_in_m = length_in_cm / 100.0;
    length_in_km = length_in_cm / 100000.0;

    cout << "The length in meter is: " << fixed << setprecision(2) << length_in_m << endl;
    cout << "The length in kilometer is: " << fixed << setprecision(2) << length_in_km << endl;

    return 0;
}

File: Basic Coding Problems/01 Fundamentals/007 Write a program to enter length in centimeter and convert it into meter and kilometer/Java/centimeter_to_meter_kilometer.java
import java.util.Scanner;

public class centimeter_to_meter_kilometer {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter the length in centimeter: ");
        int lengthInCentimeter = scanner.nextInt();

        float lengthInMeters = lengthInCentimeter / 100.0F;
        float lengthInKilometers = lengthInCentimeter / 100000.0F;

        System.out.printf("The length in meter is: %.2f\n", lengthInMeters);
        System.out.printf("The length in kilometer is: %.2f\n", lengthInKilometers);
    }
}